tests/test_dataloader.py

```diff
@@ -90,3 +91,17 @@ def test_that_dataset2_must_not_have_data(self):
             dataloader = PartitionDistributingDataLoader(
                 self.dataset1, self.dataset1, batch_size=100
             )
+
```

Comment:
Thanks for the tests. Could we also have tests for `compute_psi` in a `test_psi.py` file

Reply:
I added the test but now the GitHub checks fail because the *_psi_bindings.so* file is missing. We probably have to add the `./build-psi.sh` step to the test setup.

Reply:
Because the `PSI` repo uses bazel, i assume we don't need to do any extra setup in the test script? we should be able to call `build-psi.sh`

Reply:
Correct, executing `build-psi.sh` should put the necessary file (*_psi_bindings.so*) in the correct folder, we depend on an existing bazel installation though.